src/app/templates/js/angular/package.json

```diff
@@ -5,6 +5,7 @@
   "version": "0.1.0",
   "scripts": {
     "start": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 3000",
+    "sideload": "node node_modules/office-toolbox/app/office-toolbox.js sideload -m <%= projectInternalName %>-manifest.xml -a <%= hostInternalName %>",
```

Comment:
This can be simplified to "sideload": "office-toolbox sideload -m …" (in all of the package.json files)